pkgs/applications/printing/cups-browsed/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, cups
+, libcupsfilters
+, libppd
+, glib
+, withAvahi ? true
+, avahi
+, withLDAP ? true
+, openldap
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cups-browsed";
+  version = "2.0b2";
```

Comment:
unstable-xxxx-xx-xx?

Reply:
Good catch, I also found some reverse instances where I specified a version but fetched the current master commit instead (oops). Not a big deal though as all releases are very recent.

Reply:
resolved in rebased commits